tests/EventTest.php

```diff
@@ -91,9 +91,11 @@ public function testEventIsGeneratedWithUniqueIdentifier(): void
 
     public function testToArray(): void
     {
+        $event = new Event();
+
         $expected = [
             'event_id' => str_replace('-', '', static::GENERATED_UUID[0]),
-            'timestamp' => gmdate('Y-m-d\TH:i:s\Z'),
+            'timestamp' => $event->getTimestamp(),
```

Comment:
Changed to avoid this occasional failure: https://ci.appveyor.com/project/sentry/sentry-php/builds/29441367/job/hnjqwl6587bv6w4v

Reply:
I agree that it's a pity when it happens, but we should avoid using the getter because if it returns the date in the wrong format (it's just an example) we won't catch it as we are basically comparing that what the getter returns is what we expect. Yes, in a lot of other tests we do the same but in this specific case the format of the date is what we wanna test other than the value itself of course

Reply:
bf6d717